charts/ucmc-service/values.yaml

```diff
@@ -24,9 +24,12 @@ java:
     SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net/o/jwks
     AUTH_IDAM_CLIENT_BASEURL: https://idam-api.{{ .Values.global.environment }}.platform.hmcts.net
     AUTH_PROVIDER_SERVICE_CLIENT_BASEURL: http://rpe-service-auth-provider-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    FEES_API_URL: http://fees-register-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
+    PAYMENTS_API_URL: http://payment-api-{{ .Values.global.environment }}.service.core-compute-{{ .Values.global.environment }}.internal
   keyVaults:
     ucmc:
       resourceGroup: ucmc
       secrets:
         - ucmc-idam-client-secret
         - microservicekey-ucmc-service
+        - ucmc-service-s2s-secret
```

Comment:
I think microservicekey and s2s are the same thing, did you use the service auth provider repo to generate?

Reply:
https://github.com/hmcts/service-auth-provider-app/commit/e4ff2426d4946af3f9925164971fbff71ac9fc22